modules/aws-http4s/src/smithy4s/aws/internals/AwsEc2QueryCodecs.scala

```diff
@@ -49,7 +49,7 @@ private[aws] object AwsEcsQueryCodecs {
 
   private val addErrorStartingPath = (_: Hints).add(XmlStartingPath(List("Response", "Errors", "Error")))
   private val discriminatorReaders =
```

Comment:
`discriminatorDecoders`

I'm happy to make the changes as I do my review if you want, it will avoid comment spam